Frontend/src/test/pages/marketplace/components/product/ProductInfo.test.jsx

```
@@ -0,0 +1,137 @@
+    name: 'Test Product',
+    description: 'Fantastic product description',
+    price: 20.5,
+    originalPrice: 30.0,
+    keyFeatures: ['Feature A', 'Feature B'],
+    soldCount: 10,
+    isOrganic: true
+  };
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)